### PR TITLE
learning rate and hidden layers

### PR DESCRIPTION
**Pull request by CrookedNoob**

### CODE REVIEW

**Review by CrookedNoob — COMMENTED** [review]
okay

**Comment by CrookedNoob** [comment]
also models added with epochs and validation accuracy
